lib/scrabble.rb

```diff
@@ -10,4 +10,59 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+
+
+    if word == nil
+      return nil
+    else
+      letters = word.chars
+
+      return_score = letters.map do |score|
+        score_it = score.upcase
+        point_values[score_it]
+      end
+    end
+
+    final_score = return_score.sum
+
+    final_score
+  end
+
+  def score_with_multipliers(word, letter_multiplier, word_multiplier = 1)
+
+    if word == nil
+      return nil
+    else
+      letters = word.chars
+      word_length = word.length
+
+      return_score = letters.map do |score|
+        score_it = score.upcase
+        point_values[score_it]
+      end
+
+      x = 0
+
+      final_score = return_score.map do |score|
```

Comment:
This is a good way to keep track of the index you are on in any given iteration - for a better way, look into `map.with_index` :)